### PR TITLE
Enhancement: Use a collapsable Twitter Bootstrap navbar

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] replaces the existing navbar with one provided by Twitter Bootstrap
* [x] removes unused resources

### Before

screen shot 2015-02-24 at 16 19 49

### After

screen shot 2015-02-24 at 16 37 46

### Before

screen shot 2015-02-24 at 16 20 08

### After

screen shot 2015-02-24 at 17 24 33
screen shot 2015-02-24 at 17 24 38

### Before

screen shot 2015-02-24 at 17 26 55

### After

screen shot 2015-02-24 at 17 27 10

### CODE REVIEW

**Comment by ins0** [comment]
:+1:

**Comment by gianarb** [comment]
:+1:

File: module/Application/view/layout/layout.phtml [hunk 1]
```diff
@@ -20,39 +19,56 @@
...
-                <?php if ($userDisplayName): ?>
-                    <li class="sep">|</li>
-                    <li class="user-info">
+
+        <nav class="navbar navbar-default navbar-fixed-top">
+            <div class="container-fluid">
+                <div class="navbar-header">
+                    <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
+                        <span class="sr-only">Toggle navigation</span>
+                        <span class="icon-bar"></span>
```
Comment (Ocramius) [inline module/Application/view/layout/layout.phtml]: What are these exactly?

Comment (localheinz) [inline module/Application/view/layout/layout.phtml]: This stuff is basically the infamous :hamburger: icon right here:

screen shot 2015-02-24 at 17 16 45

**Comment by localheinz** [comment]
@Ocramius 

Shall we use a black one and use the same link colours to make the step a bit smaller?

File: module/Application/view/layout/layout.phtml (hunk 1 above)
Comment (Ocramius) [inline module/Application/view/layout/layout.phtml]: Yeah, so 3 repeated `<span/>`: comment it, maybe? :)

**Comment by Ocramius** [comment]
> Shall we use a black one and use the same link colours to make the step a bit smaller?

Not following the question, can you rephrase it?

**Comment by localheinz** [comment]
@Ocramius 

Ok, instead of the `default` navbar

screen shot 2015-02-24 at 21 01 13

shall we use an `inverse` one:

screen shot 2015-02-24 at 21 01 04

as it is closer to the current navigation?

**Comment by Ocramius** [comment]
I think this one is fine. Waiting for travis and then I'm merging :)

**Comment by Ocramius** [comment]
:+1:

**Comment by localheinz** [comment]
Thank you, @Ocramius!
